### DISCUSSION TITLE
Ferramentas para medir e melhorar performance de apps híbridos (cordova e ionic)

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#496] [Perguntas e Respostas]
Fala galera beleza?! Vocês conhecem alguma ferramenta ou forma de medir e melhorar performance de apps híbridos?

**Comment by pedrocoder** [comment]
medir  eh dificl   ,  mais melhorar performace se usa o ionic 2   compilando com AOT do angular ele vai  acrescentar e muito sua aplicacao
